ur5_example.R

```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers

```

Comment:
Please rename the file to something understandable to everybody: for example: `robot_arm_example.R`?